tests/boot/boot_from_pxe.pm

```diff
@@ -107,27 +93,25 @@ sub run {
         # 'ssh=1' and 'sshd=1' are equal, both together don't work
         # so let's just set the password here
         $cmdline .= "sshpassword=$testapi::password ";
-        type_string $cmdline, $type_speed;
+        type_string_slow $cmdline;
     }
 
     if (check_var('SCC_REGISTER', 'installation') && !(check_var('VIRT_AUTOTEST', 1) && check_var('INSTALL_TO_OTHERS', 1))) {
-        type_string(registration_bootloader_cmdline, $type_speed);
+        type_string_slow(registration_bootloader_cmdline);
     }
 
     specific_bootmenu_params;
-
     send_key 'ret';
     save_screenshot;
 
-    if ((check_var('BACKEND', 'ipmi') && !get_var('AUTOYAST')) || get_var('SES5_DEPLOY')) {
-        my $ssh_vnc_wait_time = 600;
-        assert_screen((check_var('VIDEOMODE', 'text') ? 'sshd' : 'vnc') . '-server-started', $ssh_vnc_wait_time);
+    if (check_var('BACKEND', 'ipmi')) {
```

Comment:
Check of !get_var('AUTOYAST') is missing here. It will break the original autoyast host installation tests.

Reply:
fixed